### PR TITLE
Add HW3 solution

### PR DESCRIPTION
**Pull request by jigsaff**

### CODE REVIEW

**Comment by igor-budzin** [comment]
MovieTile story doesn't log any actions but should log "onClick". The same with SortControl - should log onChange

File: src/App.js [hunk 1]
```diff
@@ -1,50 +1,115 @@
...
-import GenreSelect from './components/GenreSelect';
+import MovieTile from './components/MovieTile';
+import MovieDetails from './components/MovieDetails';
+import SortControl from './components/SortControl';
+import { useState } from 'react';
 
 function App() {
-  const genres = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
-  const selectedGenre = 'All';
+  const [sortOrder, setSortOrder] = useState('releaseDate');
```
Comment (igor-budzin) [inline src/App.js]: It is better to keep such things (I mean string "releaseDate") in separate place and do not hardcode it in the component. Data structure could look like this but is up to you
`const sortOrders = [
 {
 value: 'releaseDate',
 formatterValue: 'Release date'
},
 {
 value: 'releaseDate',
 formatterValue: 'Release date'
},
]`

**Review by igor-budzin — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (jigsaff) [reply src/App.js]: So just don't use a string "releaseDate", but use an array and have useState use that array?

Comment (igor-budzin) [reply src/App.js]: as example
image

**Pull request merged**